lms/templates/courseware_search/search_item_seq.underscore

```diff
@@ -0,0 +1,4 @@
+<div class='sri-excerpt'></div>
+<span class='sri-type'></span>
+<span class='sri-location'><%- location.join(' ▸ ') %></span>
+<a class="sri-link" href="<%- url %>"><%= gettext("View") %> <i class="icon-arrow-right"></i></a>
```

Comment:
we've upgraded font-awesome so you now need to add the following classes instead of icon-arrow-right --> "fa fa-arrow-right"